### DISCUSSION TITLE
Webmin Terminal Plugin Not Working -

### DISCUSSION THREAD

**Discussion by westerfunk** [webmin/webmin#1801] [General]
Received this when attempting to use the Terminal plugin on the latest Webmin (2.003, theme 20.10):

HTTP/1.0 500 Perl execution failed Server: MiniServ/2.003 Date: Sun, 27 Nov 2022 17:32:23 GMT Content-type: text/html; Charset=utf-8 Connection: close
ERROR — PERL EXECUTION FAILED
Undefined subroutine &xterm::allocate_miniserv_websocket called at /usr/share/webmin/xterm/index.cgi line 141.

Just wanted to make someone aware if you weren't already.

**Comment by iliaross** [comment]
Thanks. We are aware and fixed it this issue upstream. We will do our best to make a new release that will fix that problem (_asap_).

**Reply by westerfunk** [reply]
No problem at all, thanks for the quick response!
